--- mallory-0.9/admin/cvs.sh.bak	2004-03-06 17:42:43.000000000 +0100
+++ mallory-0.9/admin/cvs.sh	2006-05-26 18:38:18.000000000 +0200
@@ -64,7 +64,7 @@
     echo "*** KDE requires automake 1.6.1 or newer"
     exit 1
     ;;
-  automake*1.6.* | automake*1.7* ) : ;;
+  automake*1.6.* | automake*1.7* | automake*1.8* | automake*1.9* ) : ;;
   "" )
     echo "*** AUTOMAKE NOT FOUND!."
     echo "*** KDE requires automake 1.6.1 or newer"
